web-console/src/views/tasks-view.tsx

```diff
@@ -303,6 +303,16 @@ ORDER BY "rank" DESC, "created_time" DESC`);
         onAction: () => this.setState({ terminateSupervisorId: id })
       }
     ];
+    if (type === 'kafka' || type === 'kinesis') {
+     actions.push(
+       {
+         icon: IconNames.WRENCH,
+         title: 'Open in json spec editor',
+         onAction: () => this.props.goToLoadDataView(id)
+       });
+    }
+    // @ts-ignore
```

Comment:
what is this line?

Reply:
when i have icon: IconNames.CROSS or icon: 'cross' I get the error that string is not assignable to type of IconName